libsquoosh/src/codecs.ts

```diff
@@ -421,7 +450,7 @@ export const codecs = {
       await oxipngPromise;
       return {
         encode: (
-          buffer: Uint8Array,
+          buffer: Uint8ClampedArray | ArrayBuffer,
```

Comment:
This is more of a question than the review comment, why did we need to change this to `Uint8ClampedArray | ArrayBuffer`?

Reply:
Technically `ImageData` always gives `Uint8ClampedArray`, not sure why this wasn’t a type error before.

Reply:
This was to make sure all the `encode()` signatures are the same for each codec

Reply:
We can change this to `Uint8ClampedArray` (without the union) if we change this line:

https://github.com/GoogleChromeLabs/squoosh/blob/fafcf97f0c76f34401a9f5bb8044a4c112dd224a/libsquoosh/src/index.ts#L128

Basically from `bitmapIn.data.buffer` to `bitmapIn.data`

Reply:
I think it makes sense to do that change, then it would be nice if we can test the library by hand to make sure it doesn't break anything (I don't expect it to so but... 😅)